package.json

```diff
@@ -34,6 +34,7 @@
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1",
     "crypto-js": "^4.0.0",
+    "js-sha512": "^0.8.0",
```

Comment:
we have `[crypto-js](https://github.com/brix/crypto-js)` in our dependencies, I checked their docs and they have hashing algorithms (including sha512), can you investigate using it? to minimize our dependency tree.